pyoneer/activations/__init__.py

```diff
@@ -2,5 +2,4 @@
 from __future__ import division
 from __future__ import print_function
 
-from pyoneer.rl.envs.batch_env_impl import BatchEnv
-from pyoneer.rl.envs.process_env_impl import ProcessEnv
+from pyoneer.activations.activations_impl import swish
```

Comment:
Is there any reason for using `activations_impl`? Just curious. You could push the definition of `swish` in the `__init__.py` that lives under the `pyoneer/activations/` directory. I'm not sure about best practices on this.

Reply:
You typically don't want to define anything in an `__init__.py` file as it affects the import error. For something like importing just the `swish` function, it's not going to be a big deal, but in general, the best practice is put definitions into individual submodules within a containing module and then import them into the containing module's namespace if that's where you want them.

Reply:
Agree with Andrew. I'd also add that the naming convention comes from TensorFlow.